3_extracting_features/DPutils.py

```diff
@@ -0,0 +1,220 @@
+from importlib.resources import path
+import pandas as pd
+import pathlib
+import shutil
+
+from PIL import Image
+import os
+
+def copy_DP_files(
+    project_path: pathlib.Path,
+    config_name: str,
+    checkpoint_name: str,
+):
+    """Copy config and checkpoint files to their designated location in DP project as assigned by the project_path
+    Args:
+        project_path (pathlib.Path): Path for DP project to be located in
+        config_name (str): Name of config file to copy
+        checkpoint_name (str): Name of checkpoint file to copy
+    """
+
+    # Copy config file to DP project
+    config_load_path = pathlib.Path(f"DP_files/{config_name}")
+    config_save_path = pathlib.Path(f"{project_path}/inputs/config/{config_name}")
+    config_save_path.parents[0].mkdir(parents=True, exist_ok=True)
+    shutil.copyfile(config_load_path, config_save_path)
+
+    # Copy checkpoint file to DP project
+    checkpoint_load_path = pathlib.Path(f"DP_files/{checkpoint_name}")
+    checkpoint_save_path = pathlib.Path(
+        f"{project_path}/outputs/efn_pretrained/checkpoint/{checkpoint_name}"
+    )
+    checkpoint_save_path.parents[0].mkdir(parents=True)
+    shutil.copyfile(checkpoint_load_path, checkpoint_save_path)
+
```

Comment:
Functions usually have 2 new lines between them (PEP8 says so). Has this file been blacked?